espflash/src/command.rs

```diff
@@ -267,14 +270,14 @@ impl<'a> Command<'a> {
             Command::SpiAttach { spi_params } => {
                 write_basic(writer, &spi_params.encode(), 0)?;
             }
-            Command::ChangeBaud { speed } => {
+            Command::ChangeBaud { new_baud, prior_baud } => {
                 // length
                 writer.write_all(&(8u16.to_le_bytes()))?;
                 // checksum
                 writer.write_all(&(0u32.to_le_bytes()))?;
                 // data
-                writer.write_all(&speed.to_le_bytes())?;
-                writer.write_all(&0u32.to_le_bytes())?;
+                writer.write_all(&new_baud.to_le_bytes())?;
+                writer.write_all(&prior_baud.to_le_bytes())?;
```

Comment:
In esptool.py they check to is_stub 

if a stub is not running they still send 0 for that parameter. I suspect that may not be necessary? but i'm not 100% sure.

Reply:
Nevermind! Just saw that it was set to 0 elsewhere! Ignore my comment.